src/modules/sync/components/sync-dialog.tsx

```diff
@@ -86,14 +90,21 @@ export function SyncDialog( {
 
 	const siteEnv = getSiteEnvironment( remoteSite );
 
-	const localSiteName = localSite.name;
+	const localSiteName = (
+		<>
+			<span className="inline-block">
+				<StudioBadge className="h-6" />
+			</span>
+			<span className="text-gray-600"> { localSite.name } </span>
+		</>
+	);
 	const remoteSiteName = (
-		<div>
+		<>
 			<span className="inline-block">
-				<EnvironmentBadge type={ siteEnv } />
+				<EnvironmentBadge type={ siteEnv } className="h-6" />
 			</span>
 			<span className="text-gray-600"> { remoteSite.name } </span>
-		</div>
+		</>
 	);
```

Comment:
Can we move these two components to a separate component? I see some similarities so they could be even merged into a one single component.
Maybe we could pass the badge or siteEnv and name as props.